packages/fakta-arbeid-og-inntekt/src/components/manglendeInntektsmelding/ManglendeInntektsmeldingForm.tsx

```diff
@@ -127,7 +127,23 @@ const ManglendeInntektsmeldingForm: FunctionComponent<OwnProps> = ({
   const svgRef = useRef<SVGSVGElement>(null);
   const [openState, setOpenState] = useState(false);
   const toggleHjelpetekst = useCallback(() => setOpenState(gammelVerdi => !gammelVerdi), []);
-
+  const radioOptions = [
+    {
+      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.TarKontakt' }),
+      value: ArbeidsforholdKomplettVurderingType.KONTAKT_ARBEIDSGIVER_VED_MANGLENDE_INNTEKTSMELDING,
+    },
+    {
+      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.GåVidere' }),
+      value: ArbeidsforholdKomplettVurderingType.FORTSETT_UTEN_INNTEKTSMELDING,
+    },
+  ];
+  // Dette valget skal ikke være tilgjengelig hvis arbeidsgiver ikke er en bedrift
+  if (radData.arbeidsgiverIdent.length === 9) {
+    radioOptions.splice(1, 0, {
+      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.MeldingArbeidsgiverNavNo' }),
+      value: ArbeidsforholdKomplettVurderingType.MELDING_TIL_ARBEIDSGIVER_NAV_NO,
+    });
+  }
```

Comment:
Kanskje man kan gjøre den inline?
```suggestion
  const radioOptions = [
    {
      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.TarKontakt' }),
      value: ArbeidsforholdKomplettVurderingType.KONTAKT_ARBEIDSGIVER_VED_MANGLENDE_INNTEKTSMELDING,
    },
    ...(radData.arbeidsgiverIdent.length === 9 ? [{
      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.MeldingArbeidsgiverNavNo' }),
      value: ArbeidsforholdKomplettVurderingType.MELDING_TIL_ARBEIDSGIVER_NAV_NO,
    }]:[]),
    {
      label: intl.formatMessage({ id: 'InntektsmeldingInnhentesForm.GåVidere' }),
      value: ArbeidsforholdKomplettVurderingType.FORTSETT_UTEN_INNTEKTSMELDING,
    },
  ];
```

Reply:
Syns det blir hakke mer lesbart slik det er, så tror jeg beholder det :D Men mest preferanse